CondFormats/JetMETObjects/src/JetResolutionObject.cc

```diff
@@ -0,0 +1,437 @@
+#include <CondFormats/JetMETObjects/interface/JetResolutionObject.h>
+#include <CondFormats/JetMETObjects/interface/Utilities.h>
+#include <FWCore/Framework/interface/EventSetup.h>
+#include <FWCore/Framework/interface/ESHandle.h>
+#include <FWCore/Utilities/interface/EDMException.h>
+
+#include <CondFormats/DataRecord/interface/JetResolutionRcd.h>
+#include <CondFormats/DataRecord/interface/JetResolutionScaleFactorRcd.h>
+
```

Comment:
unneeded includes: here you do not deliver the object via `EventSetup`, you are implementing the interface of the entity classes.